BoxContentSDK/BoxContentSDK/OAuth2/BOXAuthorizationViewController.m

```diff
@@ -149,7 +150,8 @@ - (void)viewDidLoad
     } else if ([self.SDKClient.session isKindOfClass:[BOXAppUserSession class]]) {
         __weak BOXAuthorizationViewController *me = self;
         [self.SDKClient autheticateAppUserWithCompletionBlock:^(BOXUser *user, NSError *error) {
-            self.completionBlock(me, user, error);
+            [me.URLSession invalidateAndCancel];
```

Comment:
Setting `URLSession` to `nil` explicitly in `dealloc` wouldn't work in this case?